contracts/proxy/diamond/readable/DiamondReadableInternal.sol

```diff
@@ -102,23 +102,23 @@ abstract contract DiamondReadableInternal is IDiamondReadableInternal {
         uint256 selectorIndex;
 
         // loop through function selectors
-        for (uint256 slotIndex; selectorIndex < l.selectorCount; slotIndex++) {
-            bytes32 slot = l.selectorSlots[slotIndex];
+        for (uint256 slugIndex; selectorIndex < l.selectorCount; slugIndex++) {
+            bytes32 slug = l.selectorSlugs[slugIndex];
 
             for (
-                uint256 selectorSlotIndex;
-                selectorSlotIndex < 8;
-                selectorSlotIndex++
+                uint256 slugSelectorIndex;
+                slugSelectorIndex < 8;
+                slugSelectorIndex++
             ) {
                 selectorIndex++;
 
                 if (selectorIndex > l.selectorCount) {
                     break;
                 }
 
-                bytes4 selector = bytes4(slot << (selectorSlotIndex << 5));
+                bytes4 selector = bytes4(slug << (slugSelectorIndex << 5));
```

Comment:
Maybe while we are adding comments, we should add one here explaining whats happening?

Reply:
It's not necessarily out of scope, but because primary purpose of this PR is to avoid write errors, I haven't changed the read functions much except for where storage variable names have changed.  I'll add comments to this file if I have time before #248 is merged.  If not, it can be done in a future PR.